src/scss/components/switch/mixins/_hxSwitchControl.scss

```diff
@@ -0,0 +1,154 @@
+//
+// ===== Mixins for <hx-switch-control> states =====
+//
+
+// ========== PRISTINE ==========
+@mixin __switchControl($pseudo-state: null, $checkBox-state: null) {
+  @if $pseudo-state == focus {
+    box-shadow: $focus-glow;
+
+    &::-ms-check {
+      border-color: $cyan-700;
+    }
+  } @else if $pseudo-state == hover {
+    @if $checkBox-state == unchecked {
+      background-color: $gray-0;
+      border: 0.125rem solid $cyan-500;
+
+      &::before {
+        background-color: $gray-0;
+        border: 0.125rem solid $cyan-500;
+        top: 0.01rem;
+        left: 0.01rem;
+      }
+
+      &::after {
+        color: $cyan-500;
+      }
+    } @else {
```

Comment:
Please indent this block.

Reply:
As per our lint rules, `yarn lint` is not throwing any error. So, i continued the same.